composer.json

```diff
@@ -44,5 +45,10 @@
         "PHP-CSS-Parser: Fix parsing CSS selectors which contain commas <https://github.com/sabberworm/PHP-CSS-Parser/pull/138>": "https://github.com/sabberworm/PHP-CSS-Parser/commit/fa139f65c5b098ae652c970b25e6eb03fc495eb4.diff"
       }
     }
+  },
+  "autoload": {
+    "psr-4": {
+      "Amp\\AmpWP\\": "src/"
```

Comment:
See questions below about whether root namespace should be `AMP` or `Amp`. That being said, should the package be `AmpWP` or `AMP_WP`? I see WP-CLI uses `WP_CLI`. /cc @schlessera

Reply:
Discussed in https://github.com/ampproject/amp-wp/issues/3810